scripts/pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[tool.poetry]
```

Comment:
It's fine to use poetry although I've not used it a lot, we should standardize the tools across the repo

Reply:
I'm also in favor of standardization, though I think it's less important between the scripts and the validator if we plan for the validator to get spun off into its own repo at some point.